src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java

```diff
@@ -4,6 +4,7 @@
 import java.util.ArrayList;
 
 import nl.tudelft.scrumbledore.Constants;
+import nl.tudelft.scrumbledore.Logger;
```

Comment:
This import is not used, if you had run CheckStyle or PMD then you would have find that.
<review